package/win32/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ set(CPACK_NSIS_EXTRA_INSTALL_COMMANDS "
    WriteRegStr HKCR 'Applications\\\\rstudio.exe\\\\SupportedTypes' '.Rhtml' ''
    WriteRegStr HKCR 'Applications\\\\rstudio.exe\\\\SupportedTypes' '.Rpres' ''
    WriteRegStr HKCR 'Applications\\\\rstudio.exe\\\\SupportedTypes' '.Rproj' ''
+   WriteRegStr HKCR 'Applications\\\\rstudio.exe\\\\SupportedTypes' '.nb.html' ''
```

Comment:
I think the best we can do on Windows without spending an enormous amount of time with shell extensions (which must be implemented in C++ and can't be built with MingGW) is to add an open-with entry for .html, i.e.

```
WriteRegStr HKCR '.html\\\\OpenWithList\\\\rstudio.exe' '' ''
```

Users will then be able to right-click on the notebook to open it in RStudio for editing.